format/checkpoint.go

```diff
@@ -0,0 +1,78 @@
+// Copyright 2021 Google LLC. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package format provides basic support for the common checkpoint and proof
+// format described by the README in this directory.
+package format
```

Comment:
Consider whether `format.Checkpoint` is the clearest description in client code. I don't think it's wrong, but it doesn't make my heart warm and fuzzy.

Also there's https://golang.org/pkg/go/format/ but I'm not too worried about any code importing both packages :-)

Reply:
Yeah, I share the sentiment - I considered `formats/log` - `log.Checkpoint` is much nicer, but `log` collides with golang's `log` package, and you can imagine it hitting "local" `log` packages in personalities, etc.

Any ideas?

Reply:
ok, let's try out `formats/log`